### DISCUSSION TITLE
withdraw an order from an order sequence result kernel error

### DISCUSSION THREAD

**Discussion by sehb** [openTCS/opentcs#26] [Q&A] [answered]
When an order is withdrawn from the order sequence and a new one is added, it will cause the order to go into execution but the vehicle does not move

**Comment by swltr** [comment, marked as answer]
Can you be a bit more specific and provide a complete example for reproduction? You do not mention the version of openTCS you are using. Furthermore, it is not clear whether "a new one is added" means you have added a new transport order to the same order sequence or simply to the order pool or to another order sequence. Also, it is not clear which parameters you set on the transport orders and the order sequence. Such information can help to identify the problem.

As you suspect a problem with openTCS, it's better to create an issue for that ([here](https://github.com/openTCS/opentcs/issues/new/choose)) - and maybe post a link to it here.

**Reply by swltr** [reply]
For future reference, the issue @sehb created for this is #27.

**Comment by sehb** [comment]
release 5.5.0

Step 1: Create an order sequence;
Step 2: Add an order to the sequence;
Step 3: After the order goes into "BEGING_PROCESSED" , withdrawn the order;

And now, the order state is always in "BEGING_PROCESSED" state and cannot go into a "WITHDRAWN" state.
After that, the vehicle can not process any new order.
